sdk-setup/src/mb2

```diff
@@ -1888,69 +1888,70 @@ sign_rpms_native() {
 
     local extra_args=()
 
-    if [[ $OPT_PACKAGE_SIGN_USER ]]; then
-
-        if inside_build_engine; then
-            local shared_key=${OPT_SHARED_DIR}/gnupg/${OPT_PACKAGE_SIGN_USER}.key
-            if [[ ! -f $shared_key ]]; then
-                notice "Cannot sign packages: Internal error: No GPG key shared for the user '$OPT_PACKAGE_SIGN_USER'."
-                return 1
-            fi
+    if [[ ! $OPT_PACKAGE_SIGN_USER ]]; then
+        fatal "Cannot sign packages: No signing user specified."
```

Comment:
We used `notice` for all the others but now that I see this I think `fatal` would really be more accurate - if the user asks for signing, failure to do so should be fatal.